ciere/json/detail/value_impl.hpp

```diff
@@ -45,6 +45,7 @@ namespace ciere { namespace json
          static R apply( T const & v
                        , typename boost::enable_if<boost::is_convertible<R,T> >::type* dummy=0 )
          {
+            (void)dummy;
             return v;
          }
 
```

Comment:
I think it's a bad idea because it increases the probability of errors, if for some reason someone needs to use the commented arguments...

But okay, I'll create new pull request.